static/des_components/des-y3a2/des-y3a2-key-catalogs.html

```diff
@@ -0,0 +1,74 @@
+<dom-module id="des-y3a2-key-catalogs">
+
+
+<template>
+
+      <style include="shared-styles">
+
+
+</style>
+
+
+<des-card heading="Y3KP Catalogs">
+<div class="card-content">
+
+    <h2>
+      <strong>Overview</strong>
+    </h2>
+    <p>
+      <span style="color: rgb(0,0,0);">In <a class="external-link" href="http://adsabs.harvard.edu/abs/2018ApJS..235...33D" rel="nofollow">Drlica-Wagner et al. (2018)</a> we describe the creation, content, and validation of the Dark Energy Survey (DES) internal year-one cosmology data set, Y1A1 GOLD, in support of the DES Y1 cosmological analyses. Y1A1 GOLD contains parameters generated by reanalyzing pixel-level data from multi-epoch postage stamps of each object in the Y1A1 GOLD coadd catalog using the<span style="color: rgb(0,0,0);"> </span>
+        <strong>
+          <a href="https://github.com/esheldon/ngmixer">ngmix</a>
+        </strong>
+        <span style="color: rgb(0,0,0);"> code (</span>
+        <a href="http://adsabs.harvard.edu/abs/2014MNRAS.444L..25S">Sheldon 2014</a>
+        <span style="color: rgb(0,0,0);">; </span>
+        <a href="http://iopscience.iop.org/article/10.3847/1538-4357/aa704b/meta">Sheldon &amp; Huff 2017</a>
+        <span style="color: rgb(0,0,0);">; </span>
+        <a href="http://adsabs.harvard.edu/abs/2016MNRAS.460.2245J">Jarvis et al. 2016</a>
+        <span style="color: rgb(0,0,0);">)</span>. The ngmix multi-epoch, multi-band, and multi-object fit is referred to as "<strong>MOF</strong>".</span>
+    </p>
+    <p>
+      <span style="color: rgb(0,0,0);">The MOF photometry served as input for the creation of photometric redshifts and shape catalogs used in weak lensing analysis, for the redMaPPer and redMaGiC catalogs. These products, together with the PSF modeling with psfex are delivered in this page<br/>
+      </span>
+    </p>
+    <h2>
+      <strong>Y1KP Products Download</strong>
```

Comment:
@aureliocarnero This should be Y3KP?

Reply:
Correct, it should be Y3KP, but dont worry, because that page needs to be re-written completely